app/_locales/en/messages.json

```diff
@@ -1535,8 +1535,11 @@
   "swapPrivacyPolicy": {
     "message": "Swap Privacy Policy"
   },
-  "swapPrivacyPolicyDescription": {
-    "message": "Brave uses 0x (https://0x.org) as a DEX aggregator. 0x will process the Ethereum address and IP address to fulfil a transaction (including getting quotes). 0x will ONLY use this data for the purposes of processing transactions."
+  "swapPrivacyPolicyDescriptionPart1": {
```

Comment:
This can't be done for localization reasons. I'm not super worried since this is not long term but it's still not right and not ideal to split strings like that. Let's fix in a follow up.

Reply:
I managed to fix it in a way that doesn't split the string. Not pushing the commit, lest it dismisses your review. Going to create a separate PR as soon as this one is merged.

Reply:
ok thanks